emalloc.c

```diff
@@ -451,13 +451,10 @@ static void* alloc_from_meta(size_t bsize)
     return block_to_payload(b);
 }
 
-int emalloc_init()
+int emalloc_init_reserve(size_t init_size)
 {
-    for (int i = 0; i < num_exact_list; i++)
```

Comment:
The removal of `exact_block_list` initialization makes sense to me anyway since it's a static variable.